include/zephyr/bluetooth/hci.h

```diff
@@ -56,12 +56,14 @@ static inline const char *bt_hci_err_to_str(uint8_t hci_err)
   * of the parameters. Upon successful return the buffer is ready to have
   * the parameters encoded into it.
   *
+  * @deprecated Use bt_hci_cmd_alloc() instead.
+  *
   * @param opcode     Command OpCode.
   * @param param_len  Length of command parameters.
   *
   * @return Newly allocated buffer.
   */
-struct net_buf *bt_hci_cmd_create(uint16_t opcode, uint8_t param_len);
+__deprecated struct net_buf *bt_hci_cmd_create(uint16_t opcode, uint8_t param_len);
```

Comment:
Are all of this changes cherry-picks? If not, maybe we could do as github-actions asks for?

Reply:
What do you mean by do as github actions asks for?

Reply:
If you look through the "Files changed" Github-actionjs has posted comments about running clang-format